src/commands/config/starboard/remove.ts

```diff
@@ -23,8 +23,8 @@ const command: Command = {
     if (emojiSplit.length === 1) {
       isValidEmoji = true
     }
-    msg.guild.emojis.cache.forEach((e) => {
-      if (emojiSplit.includes(e.name.toLowerCase())) {
+    msg.guild?.emojis.cache.forEach((e) => {
+      if (emojiSplit.includes(e.name!.toLowerCase())) {
```

Comment:
force non-null

Reply:
chỗ này an toàn hơn là check id tại vì name có thể bị trùng, dùng `id` chứ k phải `identifier` nhé, sự khác biệt là
`id` = `585543187850985493`
`identifier` = `<a:kek:585543187850985493>`

Reply:
khúc code ở trên (dòng 18 -> 25) có thể refactor lại bằng cách dùng hàm `parseDiscordToken`, hàm này sẽ extract id của 1 value discord bất kì (channel, role, emoji, etc....) và cho mình biết luôn value vừa extract là thuộc loại gì, e.g:
```js
const {
    // value extract
    id,
    // các flag để biết đây là value loại gì
    // false
    isRole,
    // false
    isChannel,
    // true
    isUser,
    etc...
} = parseDiscordToken("<@463379262620041226>") // user hollow
```